app/cronjob/trailingTradeHelper/configuration.js

```diff
@@ -202,6 +202,74 @@ const getMaxPurchaseAmount = async (
 
   return newBuyMaxPurchaseAmount;
 };
+
+const getLastBuyPriceRemoveThreshold = async (
+  logger,
+  symbol,
+  globalConfiguration,
+  symbolConfiguration
+) => {
+  const symbolBuyLastBuyPriceRemoveThreshold = _.get(
+    symbolConfiguration,
+    'buy.lastBuyPriceRemoveThreshold',
+    -1
+  );
+
+    if (symbolBuyLastBuyPriceRemoveThreshold !== -1) {
+    logger.info(
+        { symbolBuyLastBuyPriceRemoveThreshold },
+      'Last buy threshold is found from symbol configuration.'
+    );
+        return symbolBuyLastBuyPriceRemoveThreshold;
+  }
+
+  logger.info(
+      { symbolBuyLastBuyPriceRemoveThreshold },
+    'Last Buy Threshold is set as 10. Need to calculate and override it'
+  );
+
+  let newBuyLastBuyPriceRemoveThreshold = -1;
+
+    // If old max purchase maount is -1, then should calculate maximum purchase amount based on the notional amount.
+    const cachedSymbolInfo =
```

Comment:
I still don't think you use the correct configuration for vscode. Indentation is wrong.
Install recommended extensions in the `.vscode/extensions.json`.
Then when you save, it will automatically format your code.

![image](https://user-images.githubusercontent.com/5715919/120193005-3b72e800-c25f-11eb-96a9-be750e87992b.png)